sdcflows/utils/wrangler.py

```diff
@@ -495,41 +495,71 @@ def find_estimators(
     logger.debug("Attempting fmap-less estimation")
     from .epimanip import get_trt
 
+    hits = set()  # Avoid duplicates
```

Comment:
Looking through the SyN logic, it could be a good time to start breaking up the monstrous `find_estimator()` into more digestible smaller functions.

Reply:
Yeah, I think that's true. Want me to do it in this PR?

Reply:
If it's not a big lift, isolating this (SyN) logic would be a great start